### PR TITLE
Added docos for Kubernetes Job

### PR DESCRIPTION
**Pull request by ray-nham**
Added documentations for Kubernetes Job fields.

https://github.com/OctopusDeploy/OctopusDeploy/pull/8753

### CODE REVIEW

File: docs/deployments/kubernetes/deploy-container/index.md [hunk 1]
```diff
@@ -68,10 +68,42 @@ Labels are custom key/value pairs that are assigned to Kubernetes resources. The
...
+
+This field is used when creating Kubernetes `Job` resources only.
+
+`ttlSecondsAfterFinished` is an option value to specify when the job should be cleaned up after it is executed. This is handled by the `TTL Controller`. When the TTL controller cleans up a resource, it will delete it cascadingly, that is to say it will delete its dependent objects together with it.
+
 ### Deployment strategy
 
 Kubernetes exposes two native deployment strategies: [Recreate](http://g.octopushq.com/KubernetesRecreateStrategy) and [Rolling Update](http://g.octopushq.com/KubernetesRollingStrategy). When deploying containers with this step, Octopus supports a third deployment strategy called blue/green.
 
+Deployment strategies are not applicable to Kubernetes `Job` resources.
```
Comment (harrisonmeister) [inline docs/deployments/kubernetes/deploy-container/index.md:105]:
```suggestion
:::hint
Deployment strategies are not applicable to Kubernetes `Job` resources.
:::
```

File: docs/deployments/kubernetes/deploy-container/index.md [hunk 2]
```diff
@@ -68,10 +68,42 @@ Labels are custom key/value pairs that are assigned to Kubernetes resources. The
 
 The labels are optional, as Octopus will automatically add the tags required to manage the Kubernetes resources created as part of this step.
 
+### Completions
+
+This field is used when creating Kubernetes `Job` resources only.
```
Comment (harrisonmeister) [inline docs/deployments/kubernetes/deploy-container/index.md:73]:
```suggestion
:::hint
This field is used when creating Kubernetes `Job` resources only.
:::
```

File: docs/deployments/kubernetes/deploy-container/index.md [hunk 3]
```diff
@@ -68,10 +68,42 @@ Labels are custom key/value pairs that are assigned to Kubernetes resources. The
...
+
+`completions` is an optional value to specify how many pods to be initiated, one after the other.
+
+### Parallelism
+
+This field is used when creating Kubernetes `Job` resources only.
```
Comment (harrisonmeister) [inline docs/deployments/kubernetes/deploy-container/index.md:79]:
```suggestion
:::hint
This field is used when creating Kubernetes `Job` resources only.
:::
```

File: docs/deployments/kubernetes/deploy-container/index.md [hunk 4]
```diff
@@ -68,10 +68,42 @@ Labels are custom key/value pairs that are assigned to Kubernetes resources. The
...
+
+`parallelism` is an optional value to specify how many pods should run in parallel when the job is started.
+
+### Backoff limit
+
+This field is used when creating Kubernetes `Job` resources only.
```
Comment (harrisonmeister) [inline docs/deployments/kubernetes/deploy-container/index.md:85]:
```suggestion
:::hint
This field is used when creating Kubernetes `Job` resources only.
:::
```

File: docs/deployments/kubernetes/deploy-container/index.md [hunk 5]
```diff
@@ -68,10 +68,42 @@ Labels are custom key/value pairs that are assigned to Kubernetes resources. The
...
+
+`backoffLimit` is an optional value that can be used to limit the number of pods creating continuously in case the job fails.
+
+### Active deadline seconds
+
+This field is used when creating Kubernetes `Job` resources only.
```
Comment (harrisonmeister) [inline docs/deployments/kubernetes/deploy-container/index.md:91]:
```suggestion
:::hint
This field is used when creating Kubernetes `Job` resources only.
:::
```

File: docs/deployments/kubernetes/deploy-container/index.md [hunk 6]
```diff
@@ -68,10 +68,42 @@ Labels are custom key/value pairs that are assigned to Kubernetes resources. The
...
+
+`activeDeadlineSeconds` is an optional value that determines how many seconds should the job runs. The job will be terminated if it runs longer than the given time provided in this field.
+
+### TTL Seconds After Finished
+
+This field is used when creating Kubernetes `Job` resources only.
```
Comment (harrisonmeister) [inline docs/deployments/kubernetes/deploy-container/index.md:97]:
```suggestion
:::hint
This field is used when creating Kubernetes `Job` resources only.
:::
```

**Review by harrisonmeister — COMMENTED** [review]
Added hint blocks

**Review by harrisonmeister — APPROVED** [review]
:+1:

**Comment by ray-nham** [comment]
thank you, Mark @harrisonmeister :)

**Pull request merged**